tests/lib/tools/setup_nested_hybrid_system.sh

```diff
@@ -19,6 +19,8 @@ run_muinstaller() {
     local kernel_assertion="${6}"
     local label="${7}"
     local disk="${8}"
+    local passphrase="${9}"
+    local extra_muinstaller_args="${10}"
```

Comment:
Could we just `shift` then take the whole "$@"?

Reply:
@bboozzoo  good point about `store_dir`, I was concerned at first, but turns out that passing args as `${arg}` passes them even if they are empty, so order is preserved.